src/core/public/chrome/constants.ts

```diff
@@ -31,3 +31,15 @@
 export const OPENSEARCH_DASHBOARDS_ASK_OPENSEARCH_LINK = 'https://forum.opensearch.org/';
 export const GITHUB_CREATE_ISSUE_LINK =
   'https://github.com/opensearch-project/OpenSearch-Dashboards/issues/new/choose';
+
+export const WORKSPACE_APP_ID = 'workspace';
+
+export const WORKSPACE_ID_IN_SESSION_STORAGE = '_workspace_id_';
+
+export const PATHS = {
```

Comment:
These parts was duplicated with workspace constants, could we find someway to reuse them?

Reply:
@yuye-aws I think we can just navigate to workspace root page `/` here, it's not defined now, but we will have it in the future. Let's not redefine workspace FE plugin specific variables in OSD core.

For now, we can do redirect in workspace plugin by adding
```
<Redirect from="/" to={PATHS.update} />
```